### ISSUE TITLE
[v1.2.0] test_matmul_small_brute_force_2d_Nd failed in Windows builds

### ISSUE THREAD

**Issue by peterjc123** [pytorch/pytorch#24001]
```pytb
======================================================================
FAIL: test_matmul_small_brute_force_2d_Nd (__main__.TestTorch)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_torch.py", line 8960, in test_matmul_small_brute_force_2d_Nd
    self.check_single_matmul(x, y, (o, n, p))
  File "test_torch.py", line 8895, in check_single_matmul
    self.assertTrue(expected.flags['C_CONTIGUOUS'])
AssertionError: False is not true
```

**Comment by peterjc123** [comment]
@soumith thought it maybe just because the numpy version is old.

**Comment by skrah** [comment]
I wrote that test, actually the line is only there to check that numpy and pytorch do the same thing (and the test has always passed so far).

But if some numpy versions (or MKL?) don't return a c-contiguous result there's no reason to keep this check.

**Comment by soumith** [comment]
yea it looks like on numpy v1.1 on Windows (probably a particular install) is failing that assert. we can remove it

**Comment by peterjc123** [comment]
@skrah Would you please delete them in both master and v1.2.0?

**Comment by soumith** [comment]
i've just patched v1.2.0

**Comment by skrah** [comment]
@peterjc123 Yes, for master there's #24012 now.
